test/test_client_script.py

```diff
@@ -137,6 +137,40 @@ def test_client_script_delete(self):
                 self.assertEquals(num_things, _delete.call_count)
                 _delete.reset_mock()
 
+    def test_client_script_host_ssh(self):
+        """
+        Verify use cases for the client_script ssh requests.
+        """
+        sys.argv = ['', 'host', 'ssh']
+        with contextlib.nested(
```

Comment:
I need to switch `commctl`s `.travis.yml` file to use 2.7 instead of 2.6. I'll submit that and do some clean up if I can around to it today.